### PR TITLE
added all configs for now

### PR DESCRIPTION
**Pull request by ykarmesh**
## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: habitat-lab/habitat/config/habitat/task/objectnav_velocity_control.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+defaults:
+  - task_config_base
+  - actions:
+    - velocity_stop
```
Comment (jacobkrantz) [inline habitat-lab/habitat/config/habitat/task/objectnav_velocity_control.yaml:4]: Why is an explicit stop action included? I thought stop was implied beneath a velocity threshold.

File: habitat-lab/habitat/config/habitat/task/objectnav_waypoint_control.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+defaults:
+  - task_config_base
+  - actions:
+    - velocity_stop
```
Comment (jacobkrantz) [inline habitat-lab/habitat/config/habitat/task/objectnav_waypoint_control.yaml:4]: Same for waypoints -- is stop implied beneath thresholded distance/turn values?

File: habitat-baselines/habitat_baselines/config/objectnav/ddppo_objectnav_hm3d_stretch_continuous_action.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+habitat_baselines:
+  torch_gpu_id: 0
+  tensorboard_dir: "tb"
+  video_dir: "video_dir"
+  test_episode_count: -1
+  eval_ckpt_path_dir: "data/new_checkpoints"
+  num_environments: 4
+  checkpoint_folder: "data/new_checkpoints"
+  trainer_name: "ddppo"
```
Comment (jacobkrantz) [inline habitat-baselines/habitat_baselines/config/objectnav/ddppo_objectnav_hm3d_stretch_continuous_action.yaml:27]: why not VER?

File: habitat-lab/habitat/config/habitat/task/objectnav_velocity_control.yaml (hunk 1 above)
Comment (ykarmesh) [reply habitat-lab/habitat/config/habitat/task/objectnav_velocity_control.yaml]: We changed the logic to have an explicit stop action based on feedback from Devendra

File: habitat-lab/habitat/config/habitat/task/objectnav_waypoint_control.yaml (hunk 2 above)
Comment (ykarmesh) [reply habitat-lab/habitat/config/habitat/task/objectnav_waypoint_control.yaml]: Same applies here

File: habitat-baselines/habitat_baselines/config/objectnav/ddppo_objectnav_hm3d_stretch_continuous_action.yaml (hunk 3 above)
Comment (ykarmesh) [reply habitat-baselines/habitat_baselines/config/objectnav/ddppo_objectnav_hm3d_stretch_continuous_action.yaml]: People are free to use VER if they want. I just followed the convention that this folder had

**Pull request merged**